ansible/setup.yml

```diff
@@ -12,5 +12,32 @@
   remote_user: centos
   become: yes
   roles:
+    - role: oasis-roles.kdump
+      kdump_package: kexec-tools
+      kdump_conf_path: /etc/kdump.conf
+      kdump_become_user: root
+      kdump_crash_path: /var/crash
+      kdump_core_collector_args: makedumpfile -l --message-level 1 -d 31
+      when: kdump_enable | default(true)
     - role: cluster_setup
+    - role: stackhpc.os-config
       monitor_secret: "{{ hostvars['localhost']['alaska_auth_p3_monitor'] }}"
+      os_config_content: |
```

Comment:
This seems to be copied from ansible/roles/cluster_setup/templates/clouds.yaml.j2. Can we delete that?